ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -7,6 +7,7 @@ declaration template components/ccm/schema;
 
 include 'quattor/types/component';
 include 'pan/types';
+include 'components/accounts/functions';
```

Comment:
This if for defined_group? Reading the definition from ncm-accounts, it looks like this assumes the group is created locally by quattor, i.e. it has no provision for using a group supplied by LDAP or any other NSS source. This feels overly restrictive.